src/ui/pages/NotificationDetails/components/ReceiveCredential/ReceiveCredential.tsx

```diff
@@ -57,15 +74,42 @@ const ReceiveCredential = ({
     dispatch(setNotificationsCache(updatedNotifications));
   };
 
+  const getAcdc = useCallback(async () => {
+    try {
+      setIsLoading(true);
+      const credential =
+        await Agent.agent.ipexCommunications.getAcdcFromIpexGrant(
+          notificationDetails.a.d as string
+        );
+
+      setCreDetail({
+        ...credential,
+        status: CredentialStatus.CONFIRMED,
+        credentialType: credential.s.title,
+        issuanceDate: credential.a.dt,
```

Comment:
Can you use the same type in the core? `<Omit<ACDCDetails, "status" | "credentialType" | "issuanceDate">>`

We removed those fields because they didn't make sense. There's no status because we haven't accepted it, and credentialType and issuanceDate are duplicates with other fields in `ACDCDetails`

Reply:
@Sotatek-TungNguyen2a maybe if your next PR you can refactor `ACDCDetails` to already omit these by default. So just `ACDCDetails` is used everywhere

Reply:
I can't because we are using shared to display cred detail therefore if I changed it, it maybe cause influence to another components.

I think we should update it when Martin refactor `ACDCDetails`

Reply:
`There's no status because we haven't accepted it`

It mean status should be pending right? Because this field is required in `ACDCDetails`

Reply:
Well we'll need to keep the status, I forgot you are using that to show the revoked banner on top probably.

But credentialType is `credential.s.title` - can you just use this inside that component and stop relying on `credentialType`? And same for `issuanceDate`

Reply:
Re status - I meant, pending means we've accepted the ACDC but it's not yet ready on KERIA. But here we haven't accepted it.

But anyway, it's easier to just keep it and show pending as your last commit does

Reply:
`But credentialType is credential.s.title - can you just use this inside that component and stop relying on credentialType? And same for issuanceDate`

Done.